packages/ui-core/core/src/lib/services/app/google-maps-loader.service.ts

```
@@ -7,18 +7,32 @@ export class GoogleMapsLoaderService {
-		return new Promise(async (resolve, reject) => {
-			window['__onGoogleLoaded'] = (ev) => {
+
+		return new Promise((resolve, reject) => {
+			// Prevent loading multiple times
+			if (document.querySelector(`script[src^="https://maps.googleapis.com/maps/api/js"]`)) {
+				resolve('google maps api already loaded');
+				return;
+			}
+
+			window['__onGoogleLoaded'] = () => {
+				delete window['__onGoogleLoaded'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)